data/fro/Vietor_1876_Loherains/stemma.gv

```diff
@@ -0,0 +1,62 @@
+digraph {
+alpha1[label="α1", color="grey"];
+alpha[label="α", color="grey"];
+beta[label="β", color="grey"];
+chi[label="χ", color="grey"];
```

Comment:
I think "χ" should be "ϰ" (kappa). It precedes "λ" in the Greek alphabet and would fit.